higher_health/utils.py

```diff
@@ -74,9 +74,9 @@ def save_data(data):
             ),
             "facility_destination_campus": data.get("facility_destination_campus"),
             "facility_destination_reason": data.get("facility_destination_reason"),
-            "history_obesity": data.get("history_obesity"),
-            "history_diabetes": data.get("history_diabetes"),
-            "history_hypertension": data.get("history_hypertension"),
-            "history_cardiovascular": data.get("history_cardiovascular"),
+            "history_obesity": data.get("history_obesity") or False,
+            "history_diabetes": data.get("history_diabetes") or False,
+            "history_hypertension": data.get("history_hypertension") or False,
+            "history_cardiovascular": data.get("history_cardiovascular") or False,
```

Comment:
@milton the above line should be `data.get("history_obesity", False)`

Reply:
Not quite. Since `''` is a valid value, `data.get("history_obesity", False)` returns `''` instead of `False`. e.g
```
>>> data = {'name': ''}
>>> data.get('name')
''
>>> data.get('name') or False
False
>>> data.get('name', False)
''
>>> data.get('missing_key', False)
False
```
The default kicks in on the last example where `missing_key` was not in the dict and it returns the default value.